app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -126,19 +129,24 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
      * Export data via email
      */
     public void exportData(View view) {
-        if (!Database.getDataFile(this).exists()) Toast.makeText(getApplicationContext(), "No database present.", Toast.LENGTH_SHORT).show();
-        else {
-            Uri contentUri = getUriForFile(this, ".fileprovider", Database.getDataFile(this));
-            Intent sharingIntent = new Intent(Intent.ACTION_SENDTO);
+        if (!Database.getDataFile(this).exists()){
+            Toast.makeText(getApplicationContext(), "No database present.", Toast.LENGTH_SHORT).show();
 
-            sharingIntent.setData(Uri.parse("mailto:")); // only email apps should handle this
-//            sharingIntent.putExtra(Intent.EXTRA_EMAIL, "email to send to ");
-            sharingIntent.putExtra(Intent.EXTRA_SUBJECT, "datafile cell-scanner " +  getFileTitle());
-            sharingIntent.putExtra(Intent.EXTRA_STREAM, contentUri);
+        } else {
+            String[] TO = {""};
 
-            if (sharingIntent.resolveActivity(getPackageManager()) != null) {
-                startActivity(sharingIntent);
-            }
+            Uri uri = FileProvider.getUriForFile(getApplicationContext(), "nl.nfi.cellscanner.fileprovider", Database.getDataFile(getApplicationContext()));
+
+            Intent sharingIntent = new Intent(android.content.Intent.ACTION_SEND);
+
+            sharingIntent.putExtra(Intent.EXTRA_EMAIL, TO);
```

Comment:
The to field now fills with some file reference; was that intended? maybe better to leave empty
<img width="200" src="https://user-images.githubusercontent.com/73476130/104717547-c24c7b80-5729-11eb-9226-5b07666d7f4a.png"/>